docs/dev_mode.md

```diff
@@ -33,17 +33,24 @@ The trigger is caused by any python file closing in either `open-autonomy/packag
 
 ## Debugging in the cluster
 
-When debugging deployments, it can be useful to have the option to spin up a hardhat node to enable debugging and testing of the issue within the cluster.
+When debugging deployments, it can be useful to have the option to spin up a hardhat node to enable debugging and testing of the issue within the cluster. First, build the image:
 
 ```bash
-VERSION=cluster-dev
-DEPLOYMENT_TYPE=kubernetes
-DEPLOYMENT_KEYS=deployments/keys/hardhat_keys.json
-SERVICE_ID=valory/oracle_hardhat
-
-autonomy build-image ${SERVICE_ID}
-autonomy deploy build ${DEPLOYMENT_KEYS} --kubernetes --force --dev
-autonomy deploy run --build-dir <build_dir>
+autonomy fetch valory/oracle_hardhat --local --service
+cd oracle_hardhat
+autonomy build-image
+```
+
+Now, push the image  to make it accessible for the cluster to pull it. You can get the tag from the previous command:
+```bash
+docker image push <tag>
+```
```

Comment:
Is this required? I would have thought no

Reply:
If you want to run it from the cluster yes, as the cluster pulls the image. it is probably not on the same machine

Reply:
This is dev mode ... so assumption would be it's the same machine

Reply:
I'll remove it then.

Reply:
And reverted